net/autossh/README.md

```diff
@@ -0,0 +1,30 @@
+# Autossh
```

Comment:
makes more sense to move this to pkg-descr -- we use the FreeBSD description for general direction, but if you have more to say about your plugin that is the spot since you can read it from the GUI.